ga/latest/kernel/helpers/build/features.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# (C) Copyright IBM Corporation 2023.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+if [ "$VERBOSE" != "true" ]; then
+  exec &>/dev/null
+fi
+
+set -Eeox pipefail 
+
+# Resolve liberty server symlinks and creation for server name changes
+/opt/ibm/helpers/build/configure-liberty.sh
```

Comment:
Since this section and configure-liberty.sh are not needed now, perhaps good to exclude them for now and add later with custom server name.

Reply:
Sounds good, I removed this from the features.sh files.